src/main/java/core/basesyntax/service/processor/ServiceProcessorOfShop.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service.processor;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public interface ServiceProcessorOfShop {
```

Comment:
```suggestion
public interface ServiceProcessorOfShop {
```
FruitRecordProcessor or FruitRecordService